addons/storyshots/storyshots-puppeteer/package.json

```diff
@@ -26,11 +26,14 @@
     "@storybook/router": "5.2.0-alpha.23",
     "core-js": "^3.0.1",
     "jest-image-snapshot": "^2.8.2",
-    "puppeteer": "^1.12.2",
     "regenerator-runtime": "^0.12.1"
   },
+  "optionalDependencies": {
+    "puppeteer": "^1.12.2"
```

Comment:
After this change, `yarn lint` fails for me locally: 

```
/Users/jetbrains/IdeaProjects/storybook/addons/storyshots/storyshots-puppeteer/src/index.js
  1:23  error  Unable to resolve path to module 'puppeteer'  import/no-unresolved
  1:23  error  Missing file extension for "puppeteer"        import/extensions
```

Reply:
I'll add a ignore for this!